### PR TITLE
Add Kinect USB problem solution

### PR DESCRIPTION
**Pull request by marjinal1st**
Sometimes Kinect's USB connection drops due to some power settings. There's [a fix around](https://devtalk.nvidia.com/default/topic/746596/embedded-systems/usb3-port-not-providing-continuous-power-/post/4239121/#4239121). This PR adds the solution to documentation.

### CODE REVIEW

File: _site/2015/02/01/13/index.html [hunk 1]
```diff
@@ -126,7 +126,7 @@ <h2 id="turtlebot-with-obstacle-avoidance">TurtleBot with Obstacle Avoidance</h2
 
 <div class="highlight"><pre><code class="language-sh" data-lang="sh">roslaunch turtlebot_rviz_launchers view_navigation.launch --screen</code></pre></div>
 
-<p>TurtleBot can’t reliably determine its initial pose (where it is) so <a href="/2015/02/01/12/">we’ll give it a hint</a> by using “2D Pose Estimate”. Select “2D Pose Estimate” and specify TurtleBot’s location. Leave <strong>RViz</strong> open so we can monitor its path planning. </p>
```
Comment (tfoote) [inline _site/2015/02/01/13/index.html:129]: Please don't make pure whitespace changes. It makes it a lot harder to review the PR.

Comment (tfoote) [inline _site/2015/02/01/13/index.html:129]: Actually @markwsilliman it looks like this has the generated outputs committed to the branch. You can let github render it via jekyll and not have to have the html in the repo.

Comment (marjinal1st) [inline _site/2015/02/01/13/index.html:129]: I actually thought that **_site** folder would be in **.gitignore**.

Gonna fix it in the next commit, thanks for the warning :+1:

**Comment by marjinal1st** [comment]
Reverted all the changed HTMLs to back, now only the post file is different.

**Comment by markwsilliman** [comment]
Merged.  Thanks Ahmet!

**Pull request merged**